content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,819 @@
+---
+title: "Migrating to the AWS SDK for Go V2"
+linkTitle: "Migrating"
+description: "How to migrate to the AWS SDK for Go V2 from AWS SDK for Go."
+---
+
+## Minimum Go Version
+
+The {{% alias sdk-go %}} requires a minimum version of Go 1.15. Migration from AWS SDK for Go to {{% alias sdk-go %}}
+may require you to upgrade your application by one or more Go versions. The latest version of Go can be downloaded on
+the [Downloads](https://golang.org/dl/) page. See the [Release History](https://golang.org/doc/devel/release.html) for
+more information about each Go version release, and relevant information required for upgrading.
+
+## Modularization
+
+The {{% alias sdk-go %}} has been updated to take advantage of the Go modules which became the default development mode
+in Go 1.13. A number of packages provided by the SDK have been modularized and are independently versioned and
+released respectively. This change allows for improved application dependency modeling, and will allow the SDK to
+to provide new features and functionality in a manner that follows the Go module versioning strategy.
+
+The following is are just some of the Go modules provided by the SDK:
+Module | Description
+--- | ---
+`github.com/aws/aws-sdk-go-v2` | The SDK core
+`github.com/aws/aws-sdk-go-v2/config` | Shared Configuration Loading
+`github.com/aws/aws-sdk-go-v2/credentials` | AWS Credential Providers
+`github.com/aws/aws-sdk-go-v2/ec2imds` | {{% alias service=EC2 %}} Instance Metadata Service Client
+`github.com/aws/aws-sdk-go-v2/service/<service name>` | Service Client Module
+`github.com/aws/aws-sdk-go-v2/feature/s3/manager` | {{% alias service=S3 %}} Transfer Manager
```

Comment:
Maybe put services sub modules and feature sub modules into their own lines stating that `service/` contains the modules for API clients, and `feature/` contain higher level utilities and features for working with APIs, like s3 transfer manager.